src/webui/www/private/views/rss.html

```diff
@@ -947,8 +947,8 @@
                 loadMethod: "xhr",
                 contentURL: "views/rssDownloader.html",
                 maximizable: false,
-                width: loadWindowWidth(id, 800),
-                height: loadWindowHeight(id, 650),
+                width: loadWindowWidth(id, 800, false),
+                height: loadWindowHeight(id, 650, false),
```

Comment:
Viewport size limitations disabled for RSS downloader window. The height probably doesn't need to be limited, since it appears to support scrolling, but I figured this window's sizing can be handled later.